apps/myaccount/src/components/change-password/change-password.tsx

```diff
@@ -315,6 +377,31 @@ export const ChangePassword: FunctionComponent<ChangePasswordProps> = (props: Ch
                             />
                         </Suspense>
                     </Form.Field>
+                    
+                    { passwordValidationConfig.showPasswordValidation  &&
+                        <Form.Field width={ 9 } data-testid={ `${testId}-new-password-validation-field` }>
+                            <div className="password-policy-description">                                
+                                <Icon id = "password-validation-length" className = {getIconProps("password-validation-length").class} color = {getIconProps("password-validation-length").color}/>
+                                <p>More than 8 characters</p>
+                            </div>
+                            
+                            <div className="password-policy-description">                                
+                                <Icon id = "password-validation-case" className = {getIconProps("password-validation-case").class} color = {getIconProps("password-validation-case").color} />
+                                <p>At least one uppercase and lowercase letter</p>
+                            </div>
+                            
+                            <div className="password-policy-description">                                
+                                <Icon id = "password-validation-number" className = {getIconProps("password-validation-number").class} color = {getIconProps("password-validation-number").color}/>
```

Comment:
These lines are too long. Format props into multilines. Use ESLint.

```suggestion
                                <Icon id = "password-validation-number" className = {getIconProps("password-validation-number").class} color = {getIconProps("password-validation-number").color}/>
```